src/layers/core/arc-layer/arc-layer-64-vertex.glsl.js

```diff
@@ -18,6 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+/* eslint-disable max-len */
```

Comment:
Nit: I think we have a max-len set to 100 characters in our eslint config. I'd prefer that we format our glsl code to fit within this limit. I suspect that a few carefully placed newlines would do the trick